scipy/stats/_continuous_distns.py

```diff
@@ -5821,6 +5821,23 @@ def _fitstart(self, data, args=None):
 nakagami = nakagami_gen(a=0.0, name="nakagami")
 
 
+# The function name ncx2 is an abbreviation for noncentral chi squared.
```

Comment:
copy-pasted from `distn_infrastructure.py` because it's not needed in `discrete_distns.py` like the others.